shark/iree_utils.py

```diff
@@ -158,7 +161,8 @@ def get_iree_module(module, device, input_type, args, func_name):
 def get_iree_compiled_module(module,
                              device: str,
                              frontend: str = "torch",
-                             func_name: str = "forward"):
+                             func_name: str = "forward",
+                             use_tuned_model: str = None):
```

Comment:
NIT: Can we rename `use_tuned_model` to something like `tuned_model_path` or `model_config_path`, since it's more representative of what the variable store. Ideally would have the same name in shark parser for easy readability.

Reply:
Sure, it's done.